Marlin/Configuration_adv.h

```diff
@@ -357,10 +357,10 @@
 // Steppers will shut down DEFAULT_STEPPER_DEACTIVE_TIME seconds after the last move when DISABLE_INACTIVE_? is true.
 // Time can be set by M18 and M84.
 #define DEFAULT_STEPPER_DEACTIVE_TIME 120
-#define DISABLE_INACTIVE_X true
-#define DISABLE_INACTIVE_Y true
-#define DISABLE_INACTIVE_Z true  // set to false if the nozzle will fall down on your printed part when print has finished.
-#define DISABLE_INACTIVE_E true
+#define DISABLE_INACTIVE_X false
+#define DISABLE_INACTIVE_Y false
+#define DISABLE_INACTIVE_Z false  // set to false if the nozzle will fall down on your printed part when print has finished.
+#define DISABLE_INACTIVE_E false
 
```

Comment:
> Makes M84 completely useless by default. That's not what we want.

Not at all. When I press the button either in Octoprint or Pronterface that states "Disable Motors", or if I go to the menu option on the RAMPS to "Disable Steppers", it emits an M84 and promptly disables motors. 

I compare this proposed edit to the existing Stable branch:

https://github.com/MarlinFirmware/Marlin/blob/1.0.x/Marlin/Configuration.h#L321-L326

> // Disables axis when it's not being used.
> #define DISABLE_X false
> #define DISABLE_Y false
> #define DISABLE_Z false
> #define DISABLE_E false // For all extruders
> #define DISABLE_INACTIVE_EXTRUDER true //disable only inactive extruders and keep active extruder enabled

In the stable branch, motors are NOT disabled by the timer that's configured from configuration_adv.h  ([which is set to 60 seconds](https://github.com/MarlinFirmware/Marlin/blob/1.0.x/Marlin/Configuration_adv.h#L234 ))

The converse question then becomes: Is Marlin Stable doing it wrong? Because that's the design decision made there.